### DISCUSSION TITLE
Colors are simply not responding

### DISCUSSION THREAD

**Discussion by jim1506** [tailwindlabs/tailwindcss#7754] [Help]
I am using Tailwind 3 with Laravel 8

I have detailed my question (with no reply) at 

[https://stackoverflow.com/questions/71197169/tailwind-3-color-palette-not-showing-new-colors](https://stackoverflow.com/questions/71197169/tailwind-3-color-palette-not-showing-new-colors)

I have tried with Chrome, Firefox and Opera - all the latest versions.

**Comment by adamwathan** [comment]
Hey! Can you please provide a reproduction? Without an actual GitHub project that someone can clone and run and see the issue happening it’s not possible to help I’m afraid.

**Comment by jim1506** [comment]
I am uploading a zip of a complete vanilla Laravel project to which I have only done 2 things:

added `const colors = require("tailwindcss/colors");`  to the tailwind.config.js file and added the following piece of code on the welcome page (opens by default). You will see this o a page /resources/views/welcome.blade.php.

The project was made by using composer create-project /laravel/laravel sampleProject,
Blade was then added with composer require laravel/blade --dev
Blade was then initialised using php artisan blade:install

and then npm install && npm run dev

was done.

```
<div class="w-full bg-stone-600 py-4">
                  sample div
                </div>
```

You will not the background is white. I did recompile with Mix after adding to the config file.

**Reply by jim1506** [reply]
I have tried to upload a zip of everything but I am getting a message "not included in the list". If you would care to kindly email me at jde@jimelliott.co.uk I will send it to you direct.

**Comment by jim1506** [comment]
I have also noticed that lg is not working. I have uploaded this very simple project to [https://jimelliott.co.uk/sampleProject.zip](url).

Some help will be very greatly appreciated.

**Comment by adamwathan** [comment]
The problem is that your `welcome.blade.php` file isn't importing your stylesheet. You need a `<link>` tag that links to your compiled CSS:

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/link

The stylesheet is being generated correctly (I see all the styles you say are missing in `/public/css/app.css`), it's just that you aren't actually using that stylesheet in that view.

You need this same line you have in your layout file:

```
<link rel="stylesheet" href="{{ asset('css/app.css') }}">
```

**Reply by jim1506** [reply]
Sorry  a mistake on my part in the uploading and the correct welcome.blade.php is attached with both the  css and js files included, although I see that the Laravel generated welcome page works fine with Tailwind commands without the inclusion of the css and js files.

I have made an amendment to try out the background with sm, md and lg and it completely fails. The line now reads:
`<div class="w-full sm:bg-stone-600 md:bg-blue-600 lg:bg-sky-600 py-4 text-amber-200">`

Even before I added the breakpoints the text was always coming up black, although the background was slate. Now the background is always white and the text black.

I never had such problems with version 2.


[welcome.blade.php.zip](https://github.com/tailwindlabs/tailwindcss/files/8241946/welcome.blade.php.zip)
